audit_log/tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime, timezone
+from typing import Callable
+
+import pytest
+from pytest_factoryboy import register
+
+from atv.tests.conftest import *  # noqa
+from audit_log.tests.factories import AuditLogEntryFactory
+from users.tests.conftest import *  # noqa
+
+
+@pytest.fixture
+def fixed_datetime() -> Callable[[], datetime]:
+    return lambda: datetime(2020, 6, 1, tzinfo=timezone.utc)
+
+
+@pytest.fixture(autouse=True)
+def setup_audit_logging(settings):
+    settings.ENABLE_SEND_AUDIT_LOG = True
+    settings.CLEAR_AUDIT_LOG_ENTRIES = True
+    settings.USE_X_FORWARDED_FOR = True
+    settings.ELASTICSEARCH_CLOUD_ID = "example:dXMtZWFzdC0xLmF3cy5mb3VuZC5pbyRjZWM2ZjI2MWE3NGJmMjRjZTMzYmI4ODExYjg0Mjk0ZiRjNmMyY2E2ZDA0MjI0OWFmMGNjN2Q3YTllOTYyNTc0Mw=="  # noqa: E501
```

Comment:
Does the ES library check that this is a valid ID or can we just use a small placeholder value?

Reply:
It does check the value, it tries to read some values out of that base64 encoded string, but I made it smaller.